build-binutils.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2019 The ClangBuiltLinux Authors
+# Description: Builds a standalone copy of binutils
```

Comment:
Can we drop the license from individual files and just keep the external one? Otherwise it's boilerplate for new files in the repo.

Reply:
Sure.

Reply:
And the Copyright since that's also in the LICENSE?

Reply:
df716ae4682180d6cef74386eea7c869eea3c5b0